core/src/main/java/hudson/util/Secret.java

```diff
@@ -175,8 +175,8 @@ public static Secret decrypt(@CheckForNull String data) {
         if (data.startsWith("{") && data.endsWith("}")) { //likely CBC encrypted/containing metadata but could be plain text
             byte[] payload;
             try {
-                payload = Base64.decode(data.substring(1, data.length()-1).toCharArray());
-            } catch (IOException e) {
+                payload = Base64.getDecoder().decode(data.substring(1, data.length()-1));
+            } catch (IllegalArgumentException e) {
```

Comment:
I would have preferred checked exceptions like it was before :(

Reply:
it is a RuntimeException, for me, it is fair to do not catch it and fail is the params are not valid.

Reply:
Isn't this a difference between what the two different libraries throw? If so, then the new behavior seems to match the old.

Reply:
yep, trilead throws an IOException, Base64 class throws an IllegalArgumentException, so, the behavior is the same right now.